corehq/apps/case_search/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+from corehq.apps.case_search.models import CaseSearchQueryAddition
+
+class CaseSearchQueryAdditionAdmin(admin.ModelAdmin):
+    list_display = ('name', 'id')
```

Comment:
I think you can add a `help_text` attribute on fields. 

also any reason not to show `domain` in the admin list?